lte/gateway/c/oai/tasks/sgw/sgw_handlers.c

```diff
@@ -2406,7 +2423,7 @@ int sgw_handle_nw_initiated_deactv_bearer_rsp(
       &spgw_ctxt->sgw_eps_bearer_context_information.pdn_connection, ebi);
 
     rc = gtp_tunnel_ops->del_tunnel(
-      eps_bearer_ctxt_p->paa.ipv4_address,
+      default_eps_bearer_entry_p->paa.ipv4_address,
```

Comment:
Why use the default bearer's ip address.
Use the ded bearer address, and make sure you copy the address to it at creation since we dont support multiple IPs yet

Reply:
done